### PR TITLE
RKT-1: Create index.html

### PR DESCRIPTION
**Pull request by madejsoon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container my-5">
+      <h1>Hello, world!</h1>
+      <div class="col-lg-8 px-0">
+        <hr class="col-1 my-4" />
+
+        <a href="git@github.com:madejsoon/hello-world.git" class="btn btn-primary"
```
Comment (mikolajgasior) [inline index.html]: It should be `https://github.com/madejsoon/hello-world` not the one you used.

Comment (madejsoon) [reply index.html]: look now

Comment (mikolajgasior) [reply index.html]: LGTM

**Review by mikolajgasior — APPROVED** [review]
Ready to merge!

**Pull request merged**